### ISSUE TITLE
Updated internal wiki ADR

### ISSUE THREAD

**Issue by widal001** [HHS/simpler-grants-gov#2501]
### Summary

Decide which tool we're using for the internal wiki.

### Options

- GitBook
- Confluence
- Notion
- Google drive

### Approvers

- @lucasmbrown-usds 
- Julius
- @margaretspring (& Nava team)

### Acceptance criteria

- [ ] The approvers for this decision have been identified (ideally before work on the ADR starts)
- [ ] The ADR is created and stored in `documentation/wiki/decisions/adr` with status "Accepted"
- [ ] The ADR has been reviewed and approved by the approvers listed above
- [ ] The ADR satisfies requirements that are outlined in the ADR template
- [ ] Any follow-up tickets have been created (if necessary)

**Comment by sarahknoppA6** [comment]
Wiki inventory has been completed: https://docs.google.com/spreadsheets/d/1uHWPqqWpZIPQzjZ1I1nAK4WlDo04sHR1Q31dVOBEwmc/edit?gid=0#gid=0

tl;dr- a lot of content doesn't have an immediately straightforward "yes, move this here" type answer because we have so much outdated documentation and we lack norms for the external wiki and google drive. Creating norms and the IA for doc sharing, as well as having a specific vision and goal for each will help inform where specific documents should live. I did my best with a first pass, but more discussion will be required.

**Comment by sarahknoppA6** [comment]
@margaretspring @andycochran please see above

**Comment by sarahknoppA6** [comment]
@widal001 @ericv-intuitial @margaretspring @andycochran the internal wiki ADR has been updated and is in a change request on the external wiki for review. https://app.gitbook.com/o/cFcvhi6d0nlLyH2VzVgn/s/Pm7UEzeiS1tbLCV1SFRu/~/changes/146/decisions/adr/internal-wiki-adr

**Comment by widal001** [comment]
Here's the [published version](https://wiki.simpler.grants.gov/decisions/adr/2024-11-20-internal-wiki)
